docs/api-docs/conf.py

```diff
@@ -19,26 +19,15 @@
     "sphinx.ext.intersphinx",
 ]
 
-html_theme = "sphinx_book_theme"
-
+html_theme = "furo"
 html_title = "Guppy compiler development docs"
+html_theme_options = {}
```

Comment:
Necessary or just good for explicitness? (Happy either way, but if the latter, might be good to add a comment at line end)